ddtrace/contrib/botocore/patch.py

```diff
@@ -43,12 +47,29 @@
 
 log = get_logger(__name__)
 
+
+def _parse_client_error_ignores(value):
+    # type: (Optional[str]) -> DefaultDict[str, Config._HTTPServerConfig]
+    ignores = collections.defaultdict(Config._HTTPServerConfig)
+
+    if not value:
+        return ignores
+
+    # Expected format: <operation>:<error-statuses>[,<operation>:<error-statuses> ...]
+    for part in value.split(","):
+        operation, error_statuses = part.split(":")
```

Comment:
How unlikely it is that we don't get the right format? Do we want to handle a wrong format, just in case?

Reply:
I wasn't sure, we could get the wrong format, but then this will error (maybe the message isn't clear enough), or the actual parsing into error ranges will also raise.